packages/venia-concept/src/selectors/cart.js

```diff
@@ -1,11 +1,3 @@
-export const getOrderInformation = (
-    { cart: { details: { billing_address } = {} } },
-    orderId
-) => ({
-    id: orderId,
-    billing_address
-});
```

Comment:
This function was only used once, and that one place has been deleted. This can be removed too.